### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Vaibhavipadamwar**
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -105,4 +105,4 @@ Please remember read and observe the [Code of Conduct](https://github.com/cncf/f
 the [Contributor Guide](https://kubeedge.netlify.com/docs/about/contributing/) on website
```
Comment (anvithks) [inline CONTRIBUTING.md:105]: Could you please update this to the actual link [https://docs.kubeedge.io/en/latest/contributing/contribute.html](https://docs.kubeedge.io/en/latest/contributing/contribute.html)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -105,4 +105,4 @@ Please remember read and observe the [Code of Conduct](https://github.com/cncf/f
 the [Contributor Guide](https://kubeedge.netlify.com/docs/about/contributing/) on website
 provides more detailed instructions on how to get your ideas and bug fixes seen and accepted.
 
-[kubeEdge-website-repo]: https://github.com/seattle-cloud-lab/kubeedge-website
\ No newline at end of file
+[kubeEdge-website-repo]: https://github.com/seattle-cloud-lab/kubeedge-website
```
Comment (anvithks) [inline CONTRIBUTING.md:108]: Change the URL to [https://github.com/kubeedge/website/](https://github.com/kubeedge/website/)

**Review by anvithks — CHANGES_REQUESTED** [review]
Thank you for your contribution. Suggested a couple of changes.

File: CONTRIBUTING.md (hunk 1 above)
Comment (kevin-wangzefeng) [inline CONTRIBUTING.md:105]:
```suggestion
the [Contributor Guide](docs.kubeedge.io/en/latest/contributing/contribute.html) on website

```

File: CONTRIBUTING.md (hunk 2 above)
Comment (kevin-wangzefeng) [reply CONTRIBUTING.md:108]:
```suggestion
[kubeEdge-website-repo]: https://github.com/kubeedge/website

```

**Pull request merged**